docs/how-to-use-relational-backend-storage.md

```diff
@@ -19,10 +19,10 @@ take advantage of relational storage to manage metadata.
 
 ### What kind of backend storage are supported
 
-Currently, relational backend storage supports the `JDBCBackend`, and uses `MySQL` as the
+Currently, relational backend storage supports the `JDBCBackend` and `MySQL` and `H2` are supported, `H2` is the
```

Comment:
"...backend storage supports the `JDBCBackend`, `MySQL` and `H2` are supported..."